test/dlc_test.json

```diff
@@ -0,0 +1,487 @@
+[ {
+  "testInputs" : {
+    "params" : {
+      "oracleInfo" : {
+        "publicKey" : "6b745e29cd1e8ec0cd3f3c817679689e2c55bf511a5032a46ac42123d68236b2",
+        "nonce" : "f336279279a95151a571ab441d9c5735aaa4c3080439a74f144b8a3005409dec"
+      },
+      "contractInfo" : [ {
```

Comment:
Would it be possible to have this as an array of objects with proper fields? Like:
```json
[ {
   "outcome": "7014aeda818f943c1a45e3d90db1a73b7bd3e665a96d784d82afd84ee824cea7",
   "local_payout": 0
} ]
```

It is really a pain to parse in the current form.